### ISSUE TITLE
Make [0,1]->(0,1) smoothing lambda adjustable

### ISSUE THREAD

**Issue by lvilnis** [iesl/probabilistic-hypercube-lattices#1]
Currently the smoothing on the probabilities (where we take a convex combination with the uniform distribution, 0.5) is hard coded to 0.999 and 1-0.999=.001. I'm thinking maybe since

1) the cubes could be a lot smaller probability than cones
2) mainly, we want to see if we can maximize marginals, not just conditionals, and really calibrate our probabilities

We could make this smoothing parameter a lot smaller than .001, since this makes .0005 the smallest probability we can represent. Like I don't see big problems with numerical stability if it was made like 1e-6 even, but im not sure.

Tl;DR add another hyperparameter and have it be (1-lambda) and lambda instead of hard coded

**Comment by lvilnis** [comment]
i can obviously write this patch myself I just don't want to fiddle with your code while your writing it so I'm just making issues right now.

**Comment by Lorraine333** [comment]
thx! added!
